client/js/components/toggle-show-hide.js

```diff
@@ -0,0 +1,22 @@
+import showHide from './show-hide';
+import { trackEvent } from '../utils/track-event';
+
+export default (el) => {
+  const toggle = showHide({el});
+  const trackingJson = el.getAttribute('data-tracking-info');
```

Comment:
Yea, but just store it in this file.

At the moment we use a combo of the `tracking-info` from the generic `js-track` and then hydrating with this specific system... just wondering if we chose one.